docs/Installing.asciidoc

```diff
@@ -633,6 +633,16 @@ The openQA web UI should be available on http://localhost/ now. To simply
 start openQA without enabling it permanently one can simply use `systemctl
 start` instead.
 
+== Enable automatic database backup
+An optional systemd service, `openqa-dump-db.service`, can be enabled to perform daily database backups. This service is triggered by the `openqa-dump-db.timer`.
+To enable automatic database backup, run:
+
+[source,sh]
+--------------------------------------------------------------------------------
+systemctl enable --now openqa-dump-db.timer
+--------------------------------------------------------------------------------
+Backups are stored at `/var/lib/openqa/backup`
+
```

Comment:
I haven't read the full section but should we mention that `openQA-local-db` needs to be installed before these services/timers can even work?

Reply:
I think it can create confusion. `openQA-local-db` package get automatically pulled with openQA: https://github.com/os-autoinst/openQA/blob/master/dist/rpm/openQA.spec#L118

Reply:
Agreed. If you think the user cannot easily end up with the service/timer installed but without the script, we're fine.